src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidator.java

```diff
@@ -47,7 +48,7 @@ public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, Strin
         return errors;
     }
 
-    private static boolean isCeasedDateOptional(HistoricalBeneficialOwnerDto historicalBeneficialOwnerDto, boolean isUnableToObtainAllTrustInfo) {
+    private static boolean isValidationOptional(HistoricalBeneficialOwnerDto historicalBeneficialOwnerDto, boolean isUnableToObtainAllTrustInfo) {
```

Comment:
isValidationOptional implies that all validation on this object is optional, would isCeasedDateValidationOptional be better?

Reply:
actually, might be ok, as other fields might also be optional in future